src/wallet/types.ts

```diff
@@ -173,6 +173,21 @@ export interface OutputRequestObj {
   timelock?: number | null; // output timelock
 }
 
+export interface DataScriptOutputRequestObj {
+  type: string; // output type (in this case will be 'data')
```

Comment:
Shoud we use a string-literal as type here, since we know this must always `data`?

```ts
export interface DataScriptOutputRequestObj {
  type: 'data';
  data: string;
}
```

Reply:
I like it. Done https://github.com/HathorNetwork/hathor-wallet-lib/pull/360/commits/458117681f174a09d235ec3019f84265332b14e3